read-fonts/src/tables/glyf.rs

```diff
@@ -610,7 +610,9 @@ pub fn to_path(
     fn to_f32(x: F26Dot6) -> f32 {
         x.to_f64() as f32
     }
-    const TWO: F26Dot6 = F26Dot6::from_i32(2);
+    fn midpoint(a: Point<F26Dot6>, b: Point<F26Dot6>) -> Point<F26Dot6> {
+        ((a + b).map(F26Dot6::to_bits) / 2).map(F26Dot6::from_bits)
```

Comment:
Is the `Div` impl for `F26Dot6` broken, or is it right "sometimes"? If it exists but does not produce the correct result in this case, that feels like a heck of a footgun.